src/components/ContributorThumbnail.js

```diff
@@ -12,29 +13,38 @@ const useStyles = makeStyles((theme) => ({
   thumbnailWrapper: {
     display: 'flex',
     flexDirection: 'row',
-    alignItems: 'center',
-    padding: '17px',
+    alignItems: 'flex-end',
+    padding: '13px',
     flexWrap: 'nowrap',
+    [theme.breakpoints.down('sm')]: {
+      padding: '0px',
+      marginTop: '1px',
+      marginLeft: '1.5px',
```

Comment:
Do we need pixel accuracy down to decimal places?

Reply:
If it is not allowed Then I can change it

Reply:
I just reviewed other pages  the margins are specified in decimals in some pages

Reply:
> I just reviewed other pages the margins are specified in decimals in some pages

We should clean other pages up as we get to them, but for this PR we can just clean up the decimal pixel values related to the Organizations page.  Thanks!